project/Build.scala

```diff
@@ -17,6 +17,7 @@ object Build extends Build {
       scalaVersion := "2.11.6",
       scalaVersion in ThisBuild := "2.11.6",
       organization := "com.gu",
+      fork in run := true,
```

Comment:
http://www.scala-sbt.org/0.13/docs/Forking.html

Before this change `System.exit` was terminating the JVM process `sbt` was using so needed to be restart after each run. Asking `sbt` to fork a new JVM when performing `run` commands means that when that process exits we return to `sbt` ready to run another command.